src/MOI_wrapper.jl

```diff
@@ -1,3 +1,8 @@
+# Copyright (c) 2013: Steven G. Johnson and contributors
```

Comment:
Should be 2013–2023 (the date of the last edit).

Reply:
In JuMP we've just standardized on the original year to avoid complications where we forget to update the latter year: https://github.com/jump-dev/JuMP.jl/blob/a228a0a2d598bc11ea3df4ba151423b99497ee4d/src/JuMP.jl#L1

It's mainly to be consistent with the LICENSE file: https://github.com/JuliaOpt/NLopt.jl/blob/master/LICENSE.md

Happy to make the change if you want here though.

Reply:
Fine.  In practice, at least in the US, copyright is automatic, whether you have a copyright notice or not, so my understanding is that the year of the effective copyright increments implicitly anyway; the year in the notice is just for information.   And since the copyright term is so long, the precise year won't be relevant for many decades anyway.